packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -812,6 +818,21 @@ export abstract class FluidDataStoreContext
 		checkTombstone = true,
```

Comment:
We might want to rename this function... can skip it for this PR if you want though

Reply:
Definitely will skip for now, we should do it in a separate PR